### PR TITLE
Updated param doc

### PR DESCRIPTION
**Pull request by nyurik**
Change-Id: I51dd93357c8b0370b3482ba60c3cf5fa6a1af217

### CODE REVIEW

File: specs/mediawiki_v1_graphoid.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ paths:
           required: true
         - name: revision
           in: path
-          description: The page revision
+          description: The page revision, or 0 if unknown
```
Comment (gwicke) [inline specs/mediawiki_v1_graphoid.yaml]: Would ', or 0 for the latest' be more accurate?

Comment (nyurik) [inline specs/mediawiki_v1_graphoid.yaml]: sure, either-or

Comment (gwicke) [inline specs/mediawiki_v1_graphoid.yaml]: I should have said precise perhaps. It's good to be explicit about which revision is returned when 0 is passed in.

**Comment by d00rman** [comment]
If the revision is optional, IMHO the route should be `/page/png/{title}{/revision}/{id}`

**Comment by nyurik** [comment]
I would rather not - revision is preferred, plus the URL is generated by
applying real values to the format string - so it will always be in the
same format.

**Pull request merged**